src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -35,10 +35,10 @@ const styles = {
   }
 }
 
-const AclInput = ({ value, onChange, disabled, maxAccessLevel, autoFocus, ...props }) => {
+const AclInput = ({ value, onChange, disabled, maxAccessLevel, isAzureWorkspace, autoFocus, ...props }) => {
   const { accessLevel, canShare, canCompute } = value
   return div({ style: { display: 'flex', marginTop: '0.25rem' } }, [
-    div({ style: { width: 200 } }, [
+    div({ style: { width: isAzureWorkspace ? 425 : 200 } }, [
```

Comment:
Can it just not take up the available space (instead of hardcoding 425)?

Reply:
@cahrens I tried having it take up available space, but the amount of available space is different for your own role than for others, which didn't look great. That's why I opted to hardcode the width for Azure like we do for GCP. I could work around it if you think it'd be best to avoid the hardcoding though.
![Screen Shot 2022-10-04 at 3 27 23 PM](https://user-images.githubusercontent.com/19961550/193908548-1baddb6f-858a-47a3-a564-6669a71195b8.png)

Reply:
This is fine, thanks!